clients/src/main/java/org/apache/kafka/common/metrics/stats/SampledStat.java

```diff
@@ -108,6 +108,7 @@ protected void purgeObsoleteSamples(MetricConfig config, long now) {
         public long eventCount;
         public long lastWindowMs;
         public double value;
+        public double secondValue;
```

Comment:
Instead of doing this, it may be better to have a subclass for the case where there are two values. Also, `reset` would have to reset `secondValue` too.